tap_twitter_ads/sync.py

```diff
@@ -262,6 +263,9 @@ def sync_endpoint(client,
         # API Call
         cursor = get_resource(stream_name, client, path, new_params)
 
+        # Cursor for children to retrieve parent_ids
+        cursor_child = copy.deepcopy(cursor)
```

Comment:
Why do we need to copy this? I can't see any changes for the cursor.

Reply:
Here, the `cursor` is `generator(yield)`. First, it will be iterated for the parent stream with the parent's bookmark. But, for the child also we want to iterate through all parent records based on the child bookmark and collect `parent_ids`. That's why we are making a `cursor` copy before the parent iteration.